recipes/mdformat-tables/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mdformat-tables" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mdformat_tables-{{ version }}.tar.gz
+  sha256: 3024e88e9d29d7b8bb07fd6b59c9d5dcf14d2060122be29e30e72d27b65d7da9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
```

Comment:
It looks like you need to add `flit_core >=2,<4` to the `host` section.

Reply:
yep... it would be great if grayskull got better at `build-system` parsing...